### DISCUSSION TITLE
Thanks for Increase Maximum Records Returned to 10K!

### DISCUSSION THREAD

**Discussion by dgold88** [FEMA/openfema-samples#7] [General]
FEMA  (@waptug and @lukelamar) ,

Thanks for increasing the maximum numbers of records returned to 10,000. This makes the API much more powerful and accessible for less sophisticated users (like myself) by decreasing the for more complex paging scripts. Well done!

David

**Comment by lukelamar** [comment]
@dgold88 
Glad it helps! We may increase it in the future depending on the observed impact to our system. We always welcome feedback and suggestions for improvement.

OpenFEMA Team
